package.json

```
@@ -10,7 +10,8 @@
         "start": "yarn workspace web-app dev",
         "lint": "eslint . --ext .js,.ts",
         "prettier": "prettier -c .",
-        "prettier:write": "prettier -w ."
+        "prettier:write": "prettier -w .",
+        "copy:contract-artifacts": "npx ts-node scripts/copy-contract-artifacts.ts"
```

```suggestion
"copy:contract-artifacts": "ts-node scripts/copy-contract-artifacts.ts"
```
`npx` shouldn't be necessary.